packages/tradershub/src/routes/Router.tsx

```diff
@@ -18,8 +19,12 @@ declare module 'react-router-dom' {
 const Router = () => {
     return (
         <Switch>
-            <Route component={CompareAccountsScreen} path={`${prefix}/compare-accounts`} />
-            <Route component={TradersHubRoute} path={prefix} />
+            <AppContainer className='max-w-[800px] lg:max-w-[1440px] lg:py-0'>
+                <Route component={CompareAccountsScreen} path={`${prefix}/compare-accounts`} />
+            </AppContainer>
+            <AppContainer>
+                <Route component={TradersHubRoute} path={prefix} />
+            </AppContainer>
```

Comment:
Reason behind this is because compare accounts is not bound to the AppContainer base styles. Therefore reused the container for its own styles.